config/amip_configs/amip.yml

```diff
@@ -9,8 +9,7 @@ dt_save_to_sol: "30days"
 dz_bottom: 30.0
 energy_check: false
 h_elem: 16
-hourly_checkpoint: true
-hourly_checkpoint_dt: 720
+checkpoint_dt: "720hours"
```

Comment:
I find it's easier to parse the YML files when we keep them alphabetized - Could you move the `checkpoint_dt` args up in the YML files that have it now?